.eslines.json

```diff
@@ -0,0 +1,8 @@
+{
+    "rulesNotToDowngrade": ["no-unused-vars"],
```

Comment:
Can you explain this option? I tried referencing the [project documentation](https://github.com/Automattic/eslines) but it seems either unnecessary in that we should leverage errors as defined in our configuration or invalid in that it's assuming that all errors are warnings if not modified in the current changeset.